topics/synthetic-biology/tutorials/basic_assembly_analysis/workflows/Genetic_Design_BASIC_Assembly.ga

```diff
@@ -0,0 +1,413 @@
+        "7": {
+            "annotation": "DNA assembly using BASIC on OpenTrons.",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/tduigou/dnabot/dnabot/3.1.0",
+            "errors": null,
+            "id": 7,
+            "input_connections": {
+                "adv|default_settings_file": {
+                    "id": 4,
+                    "output_name": "output"
+                },
+                "construct_file": {
+                    "id": 6,
+                    "output_name": "Constructs"
+                },
+                "plate_files": [
+                    {
+                        "id": 6,
+                        "output_name": "User parts plate"
+                    },
+                    {
+                        "id": 6,
+                        "output_name": "Biolegio plate"
+                    }
+                ]
+            },
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool DNA-Bot",
+                    "name": "adv"
+                },
+                {
+                    "description": "runtime parameter for tool DNA-Bot",
+                    "name": "construct_file"
+                },
+                {
+                    "description": "runtime parameter for tool DNA-Bot",
+                    "name": "plate_files"
+                }
+            ],
+            "label": "DNA-Bot",
+            "name": "DNA-Bot",
+            "outputs": [
+                {
+                    "name": "dnabot_scripts",
+                    "type": "tar"
+                }
+            ],
+            "position": {
+                "bottom": 728.8000030517578,
+                "height": 194.8000030517578,
+                "left": 1073,
+                "right": 1273,
+                "top": 534,
+                "width": 200,
+                "x": 1073,
+                "y": 534
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/tduigou/dnabot/dnabot/3.1.0",
+            "tool_shed_repository": {
+                "changeset_revision": "de6812daba39",
+                "name": "dnabot",
+                "owner": "tduigou",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"adv\": {\"default_settings_file\": {\"__class__\": \"RuntimeValue\"}, \"etoh_well\": \"A11\", \"soc_column\": \"1\"}, \"construct_file\": {\"__class__\": \"RuntimeValue\"}, \"plate_files\": {\"__class__\": \"RuntimeValue\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": null,
+            "type": "tool",
+            "uuid": "c09c99e4-5e0c-4c04-9f9b-3c30183d3816",
+            "workflow_outputs": [
+                {
+                    "label": null,
+                    "output_name": "dnabot_scripts",
+                    "uuid": "ce5be4f0-48fb-45f0-84ad-15348645c950"
+                }
+            ]
+        }
+    },
+    "tags": [
```

Comment:
aha, remove this `tags`, it is overriding the one above

Reply:
ok, done.